flink-runtime/src/test/java/org/apache/flink/runtime/io/network/buffer/NetworkBufferPoolTest.java

```diff
@@ -757,7 +746,7 @@ public void testBlockingRequestFromMultiLocalBufferPool()
             // wait until all available buffers are requested
             while (segmentsRequested.size() + segments.size() + exclusiveSegments.size()
                     < numBuffers) {
-                Thread.sleep(100);
```

Comment:
Is this a necessary change? Is it in the right commit?

Reply:
Not necessary at all. I just notice that 100 ms is too high here considering we have a loop. But perhaps, it is indeed not a good idea to have these changes in this commit. I can remove this change at all.

Reply:
We can leave it the way it is. Just wanted to check.